homeassistant/components/vizio/__init__.py

```diff
@@ -11,21 +14,18 @@
 from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.typing import ConfigType, HomeAssistantType
 
-from .const import (
-    CONF_VOLUME_STEP,
-    DEFAULT_DEVICE_CLASS,
-    DEFAULT_NAME,
-    DEFAULT_VOLUME_STEP,
-    DOMAIN,
-)
+from .const import CONF_VOLUME_STEP, DEFAULT_NAME, DEFAULT_VOLUME_STEP, DOMAIN
+
+DEFAULT_DEVICE_CLASS = DEVICE_CLASS_TV
+ICON = {DEVICE_CLASS_TV: "mdi:television", DEVICE_CLASS_SPEAKER: "mdi:speaker"}
```

Comment:
Why did we move the icon constant?

Reply:
I was mistakenly under the impression that `const.py` shouldn't have imports but I see in other integrations that it does. I will move all constants out of `__init__` into `const.py` including VIZIO_SCHEMA so that they are all in one place

Reply:
Same with the dict in `media_player`